### PR TITLE
modify HaaSPython's brief introduction, fix #1772

### LINKED ISSUE #1772: [Bug Report]: HaaS Python描述有些不合理
### 联系方式

_No response_

### 您发现了什么问题

A bug happened!
运行环境:
固件版本:
下面的文档中的HaaS Python描述不够清楚
https://github.com/alibaba/AliOS-Things/blob/master/haas_lib_bundles/python/README.md

### 分支

master (Default)

### 请选择您发现此问题的解决方案名称

_No response_

### 日志

_No response_

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
auto build success,number:198

**Comment by skylarCai** [comment]
autotest fail,number:1534

**Review by wushengtws — APPROVED** [review]

File: haas_lib_bundles/python/README.md [hunk 1]
```diff
@@ -1,86 +1,5 @@
 # HaaS Python
```
Comment (skylarCai) [inline haas_lib_bundles/python/README.md:1]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**